src/components/dialogs/parameters/short-circuit-parameters.js

```diff
@@ -56,181 +66,161 @@ export const useGetShortCircuitParameters = () => {
-                            headerId: 'paramsRetrievingError',
+                            headerId: 'invalidateShortCircuitStatusError',
                         });
                     });
-            })
-            .catch((error) => {
-                snackError({
-                    messageTxt: error.message,
-                    headerId: 'paramsChangingError',
+                    //used to update isDirty after submit
+                    reset({}, { keepValues: true });
+                })
+                .catch((error) => {
+                    setShortCircuitParams(oldParams);
+                    snackError({
+                        messageTxt: error.message,
+                        headerId: 'paramsChangingError',
+                    });
                 });
+        },
+        [
+            setShortCircuitParams,
+            shortCircuitParams,
+            snackError,
+            studyUuid,
+            reset,
+        ]
+    );
+
+    // when ever voltage profile mode change we need to reset all parameters
+    const resetAll = useCallback(
+        (initialVoltageProfileMode) => {
+            setValue(SHORT_CIRCUIT_WITH_FEEDER_RESULT, true, {
+                shouldDirty: true,
             });
-    }, [studyUuid, snackError, setShortCircuitParams]);
+            setValue(SHORT_CIRCUIT_WITH_LOADS, false, { shouldDirty: true });
+            setValue(SHORT_CIRCUIT_WITH_VSC_CONVERTER_STATIONS, true, {
+                shouldDirty: true,
+            });
+            setValue(SHORT_CIRCUIT_WITH_SHUNT_COMPENSATORS, false, {
+                shouldDirty: true,
+            });
+            setValue(SHORT_CIRCUIT_WITH_NEUTRAL_POSITION, false, {
+                shouldDirty: true,
+            });
+            const initalVoltageProfileMode =
+                initialVoltageProfileMode === INITIAL_TENSION.NOMINAL
+                    ? INITIAL_TENSION.NOMINAL
+                    : INITIAL_TENSION.CONFIGURED;
+
+            setValue(
+                SHORT_CIRCUIT_INITIAL_VOLTAGE_PROFILE_MODE,
+                initalVoltageProfileMode,
+                {
+                    shouldDirty: true,
+                }
+            );
+            setValue(
+                SHORT_CIRCUIT_PREDEFINED_PARAMS,
+                initalVoltageProfileMode,
+                {
+                    shouldDirty: true,
+                }
+            );
+        },
+        [setValue]
+    );
 
     return (
-        <>
-            <Grid
-                container
-                key="shortCircuitParameters"
-                sx={styles.scrollableGrid}
-            >
-                <BasicShortCircuitParameters
-                    shortCircuitParams={shortCircuitParams || {}}
-                    commitShortCircuitParams={commitShortCircuitParameter}
-                />
+        <FormProvider validationSchema={formSchema} {...formMethods}>
+            <Grid container paddingTop={1} paddingBottom={1}>
```

Comment:
you can shorthand this to 
```suggestion
            <Grid container padding={'8px 0'}>
```

Reply:
How  padding={'8px 0'} is equivalent to paddingTop={1} paddingBottom={1} ?
I prefer paddingTop={1} paddingBottom={1}  then hard coding px.

Reply:
padding(x) = padding-top(x) padding-right(x) padding-bottom(x) padding-left(x) = padding-top-bottom(x) padding-right-left(x) = padding-top(x) padding-left-right(x) padding-bottom(x)

source: https://developer.mozilla.org/en-US/docs/Web/CSS/padding

so padding={'8px 0'} is equivalent to paddingTop={1} paddingBottom={1} (as 1 = 8px)

but if you prefer 1 than 8px, it is ok